src/components/User/ProjectsContainer.vue

```diff
@@ -12,7 +12,8 @@ defineProps<Props>()
 <template>
   <section :class="$style.section">
     <h2 :class="$style.title">参加プロジェクト</h2>
-    <project-list :projects="projects" />
+    <project-list v-if="projects.length > 0" :projects="projects" />
+    <p v-else>参加プロジェクトはありません</p>
```

Comment:
「これまでに」を入れることでちょっとだけ表現を緩めてみるのはどうでしょうか…？
「ない」と言い切るより、「今のところは」という印象があって優しい表現の気がします
```suggestion
    <p v-else>これまでに参加したプロジェクトはありません</p>
```